inspirehep/config.py

```diff
@@ -170,53 +175,117 @@ def _(x):
     "delete_permission_factory_imp": deny_all,
     "list_permission_factory_imp": allow_all,
 }
-LITERATURE_ARXIV = deepcopy(LITERATURE)
-LITERATURE_ARXIV.update(
```

Comment:
aren't `arxiv` and `doi` only literature? so we could keep using `LiteratureRecord` maybe.